services/tests/tweets-mock.js

```diff
@@ -0,0 +1,62 @@
+export const tweetsMock = [
```

Comment:
I believe we already have tweets mock data. We should reference the existing file rather than duplicate it.

Reply:
In this case, we already have the tweets mock data, but not the result of the twitter service's `getTweets`. This new mock is the expected result of the generated data from this service.

Reply:
ah gotcha. that makes sense.